assets/javascripts/discourse/components/param-input-form.gjs

```diff
@@ -158,6 +134,9 @@ export default class ParamInputForm extends Component {
   form = null;
 
   promiseNormalizations = [];
+  formLoaded = new Promise((res) => {
+    this.__form_load_callback = res;
+  });
```

Comment:
Is this used to show existing errors of previous params right when loading the form without any user interaction?

Reply:
Yes. It may can also be done directly with setTimeout 0, I'm not sure which is better